packages/manager/src/features/IAM/hooks/adapters/accountGrantsToPermissions.ts

```diff
@@ -0,0 +1,112 @@
+import type {
+  AccountAdmin,
+  GlobalGrantTypes,
+  GrantLevel,
+} from '@linode/api-v4';
+
+/** Map the existing Grant model to the new IAM RBAC model. */
+export const accountGrantsToPermissions = (
+  globalGrants?: Record<GlobalGrantTypes, boolean | GrantLevel>
+): Record<AccountAdmin, boolean> => {
+  return {
+    // AccountAdmin
+    accept_service_transfer: Boolean(globalGrants?.account_access),
+    acknowledge_account_agreement: Boolean(globalGrants?.account_access),
+    answer_profile_security_questions: Boolean(globalGrants?.account_access),
+    cancel_account: Boolean(globalGrants?.cancel_account),
+    cancel_service_transfer: Boolean(globalGrants?.account_access),
+    create_profile_pat: Boolean(globalGrants?.account_access),
+    create_profile_ssh_key: Boolean(globalGrants?.account_access),
+    create_profile_tfa_secret: Boolean(globalGrants?.account_access),
+    create_service_transfer: Boolean(globalGrants?.account_access),
+    create_user: Boolean(globalGrants?.account_access),
```

Comment:
From my understanding `account_access` can be `null`, `read_only`, or `read_write`.


If a user has `account_access: "read_only"`, this `Boolean` would return true, but I don't think a restricted user with read_only account access can create users. 


I'm wondering if some of these should be: 
```suggestion
    create_user: globalGrants?.account_access === 'read_write',
```

Reply:
Nice catch @bnussman-akamai, yes I need to check the mapping...in progress

Reply:
Updated the mapping.